backend_api/README.md

```diff
@@ -16,87 +16,72 @@ Create your App Engine application by typing:
-
-    gcloud iam service-accounts create example \
-    --display-name "My Service Account"
+Get the project ID and save it into an environment variable
 
-Give your newly created Service Account appropriate permissions:
+    export PROJECT_ID=$(gcloud config get-value core/project)
 
-    gcloud projects add-iam-policy-binding ${PROJECT_ID} \
-    --member serviceAccount:example@${PROJECT_ID}.iam.gserviceaccount.com \
-    --role roles/owner
 
-After creating your Service Account, create a Service Account key:
+Get the key.json for the App Engine service account:
 
     gcloud iam service-accounts keys create ~/key.json --iam-account \
-    example@${PROJECT_ID}.iam.gserviceaccount.com
-
-Set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to point to where
-you just put your Service Account key:
-
-    export GOOGLE_APPLICATION_CREDENTIALS="/home/${USER}/key.json"
+    ${PROJECT_ID}@appspot.gserviceaccount.com
 
-**IMPORTANT:** Keep this key.json a secret. You should not commit this file ever.
+<span style="color:red">
+<h1>
+<b>
+IMPORTANT: Keep this key.json a secret. You should not commit this file ever.
+</b>
+</h1>
+</span>
 
-## Getting the sample code
+## Running the Backend on Cloud Shell
 
-Run the following command to clone the Github repository:
+Run the following command to clone the Github repository to your cloud shell:
 
     git clone https://github.com/Jiaxen/sample-gcp-nlp-flask.git
 
 Change directory to the backend directory:
 
     cd sample-gcp-nlp-flask/backend_api
 
-## Running "locally"
-To run our app "locally" through the cloud shell (that is, not deploying it just yet), we should create a virtual 
-environment. This is just an environment where we install the specific dependencies needed by the project and
-can run the code.
+## Running Flask on Cloud Shell
+To run our app through the cloud shell (that is, not deploying it just yet), we should create a virtual environment. This is just an environment where we install the specific dependencies needed by the project and can run the code.
+
+Install virtualenv:
+
+    pip install virtualenv
 
 Create a virtual environment and install dependencies:
 
     virtualenv -p python3 env
 
 If you do `ls` now, you will see an `/env` folder created which contains the virtual environment. 
 
-Start your virtual environment:
+Start your virtual environment (mac and linux):
 
     source env/bin/activate
 
 Install the dependencies:
 
     pip install -r requirements.txt
 
-Start your application locally using your virtual environment:
+Start your application via cloud shell using your virtual environment:
 
     python main.py
 
-Visit the link generated ('Running on http://someipaddress:8080/') to view your application running locally. Test it out!
+Visit the link generated ('Running on http://127.0.0.1:8080/') to view your application running locally. Test it out! (click on the link from cloud shell)
```

Comment:
Could we elaborate on how to test it out (might be that some of the teams wont know how to use swagger ui). Along the lines of "the post request does xyz for our use case. Click it, click 'try it out', type in a sentence, and click 'execute'. The get request does abc..."